src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const list = document.querySelector('tbody').children;
+const head = document.querySelector('thead').children[0];
+
+head.addEventListener('click', sort);
+
+function sort(header) {
+  const index = [...head.children].indexOf(header.target);
+
+  const sortedList = [...list].sort((a, b) => {
+    const rowA = a.children[index].innerText;
+    const rowB = b.children[index].innerText;
+
+    switch (header.target.innerText) {
+      case 'Name':
```

Comment:
Name, Position, Age and Salary are 'magic words', so make sense to move them to the constants